sql/query/q.sql

```diff
@@ -0,0 +1,17 @@
+-- name: GetDomains :many
+SELECT * FROM domains;
+
+-- name: PrepareForSend :many
+UPDATE sending_pool_emails AS sp
+    SET status = 'scheduled'
+    FROM (
+            SELECT id FROM sending_pool_emails
+            WHERE scheduled_time <= NOW() and status = 'scheduled'
+            ORDER BY RANDOM()
+            LIMIT $1
+        ) AS t
+    WHERE sp.id = t.id
+    RETURNING sp.*;
```

Comment:
@kandros look at this query handled with gqlc!

The corresponding code in gorm involves a transaction and does not handle random ordering

https://github.com/gyozatech/kannon/blob/8b8c6abf433b4b859c9c842afe4b4f51fb07f888/internal/pool/pool.go#L62-L91